README.md

```diff
@@ -1,127 +1,130 @@
+- **Immutable state**. That means that the store contains a state which is a one-level object with immutable attribute values. The only way to mutate the state is to emit an action, an object describing what happened.
+- **State changes are made via pure functions**. Reducers are just pure functions that take the previous state and an action, and return the next state. Many reducers can make modifications through different portions of the state in different reducers, so we avoid chaining actions in our components and we avoid triggering actions inside our stores/reducers.It also contains the 3
+When writing component based applications, it is enormously helpful to use Facebook's Flux architecture. It truly complements ReactJS' unidirectional data flow model. Facebook's Flux library provides a Dispatcher, and some examples of how to write Actions and Stores. However, there are no helpers for Action & Store creation, and Stores require 3rd party eventing.
 
-## Demo
+Flaxs is a library that provides all 3 components of Flux architecture, using Facebook's Dispatcher, and providing factories for Actions & Stores.
 
-Check out this JSFiddle Demo to see how McFly can work for you:
+### Forked from
 
-[http://jsfiddle.net/6rauuetb/](http://jsfiddle.net/6rauuetb/)
+*Flaxs* was created initially as a fork of [McFly](https://github.com/kenwheeler/mcfly).  
+Thanks to [kenwheeler](https://github.com/kenwheeler) for letting this project to happen.
 
-## Download
 
-McFly can be downloaded from:
+## Examples
 
-[http://kenwheeler.github.io/mcfly/McFly.js](http://kenwheeler.github.io/mcfly/McFly.js)
+You can review in detail our [examples guideline page](./tree/master/examples/README.md):
+
+## Demo Page
+
+Right now, we don't have a demo page, nonetheless we have an examples folder which will allow us to explain in detail the potential of Flaxs.
 
 ## Dispatcher
 
-McFly uses Facebook Flux's dispatcher. When McFly is instantiated, a single dispatcher instance is created and can be accessed like shown below:
+Flaxs uses Facebook Flux's dispatcher. When Flaxs is instantiated, a single dispatcher instance is created and can be accessed like shown below:
 
 ```javascript
-var mcFly = new McFly();
+import { flaxs } from 'flaxs';
 
-return mcFly.dispatcher;
+return flaxs.dispatcher;
 ```
-In fact, all created Actions & Stores are also stored on the McFly object as `actions` and `stores` respectively.
+In fact, all created Actions & Stores are also stored on the Flaxs object as `actions` and `stores` respectively.
 
 ## Stores
 
-McFly has a **createStore** helper method that creates an instance of a Store. Store instances have been merged with EventEmitter and come with **emitChange**, **addChangeListener** and **removeChangeListener** methods built in.
+**Stores in Flaxs are used to preserve compatibility with McFly approach, but are not recommended to use for new projects.**
```

Comment:
Even though I agree with this for short-term backwards compatibility, we should consider a roadmap in which it is deprecated and then killed-off. But that's a dicussion for another time 😄